test/test_sparse.py

```diff
@@ -3210,8 +3213,13 @@ def sparse_log(x):
     # TODO: Check after why ROCm's cusparseXcsrgemm2Nnz function doesn't return the same nnz value as CUDA
     @skipIfRocm
     @coalescedonoff
-    @dtypes(torch.double)
-    @dtypesIfCPU(torch.double, torch.cdouble)
+    @dtypes(*get_all_complex_dtypes(),
+            *get_all_fp_dtypes(include_half=False, include_bfloat16=False))
+    @dtypesIfCUDA(*(get_all_complex_dtypes() if CUDA11OrLater else ()),
+                  *get_all_fp_dtypes(
+                      include_half=(CUDA11OrLater and SM53OrLater),
+                      include_bfloat16=(CUDA11OrLater and SM80OrLater)))
+    @precisionOverride({torch.bfloat16: 1e-2, torch.float16: 1e-2})
```

Comment:
Why 1e-2 for float16 as opposed to 1e-3?

Reply:
I found that many tests use 1e-2 and none uses 1e-3.
I tried 1e-3 here and tests fail.
```python
AssertionError: False is not true : Sparse tensor values failed to compare as equal! With rtol=0.001 and atol=0.001, found 1 element(s) (out of 9) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 0.001708984375 (-0.121826171875 vs. -0.1201171875), which occurred at index 6.
```